packages/framer-scroller/plugins/GalleryThumbnailNavigation.tsx

```diff
@@ -0,0 +1,15 @@
+import type { PluginProps } from '@graphcommerce/next-config'
+import type { SidebarGalleryProps } from '@graphcommerce/next-ui/FramerScroller/SidebarGallery'
+import { useCallback } from 'react'
+import { ScrollerThumbnails } from '../components/ScrollerThumbnails'
+
+export const component = 'SidebarGallery'
+export const exported = '@graphcommerce/next-ui'
```

Comment:
~~The location of this plugin is wrong, next-ui is dependent on framer-scroller, so framer-scroller can not create a plugin on next-ui. Just directly use it in SidebarGallery?~~

Already implemented in SidebarGallery, so can be removed